File: Codigos/Aula 4/Python/scr/View/ExibicaoView.py
class ExibicaoView:

    @staticmethod
    def exibir_lista(lista: list[int], frase: str) -> None:
        print(frase)
        for item in lista:
            print(item)
        print("--------------------------")
        print(f"Total de registros: {len(lista)}")

    @staticmethod
    def exibir_tempo_execucao(rotina: int, tempo_ms: float) -> None:
        print(f"Tempo (ms) rotina {rotina}: {tempo_ms:.2f}")

File: Codigos/Aula 4/Java/scr/View/ExibicaoView.java
package projeto.view;

import java.util.List;

public class ExibicaoView {

    public static void exibirLista(List<Integer> lista, String frase) {
        System.out.println(frase);
        for (Object item : lista) {
            System.out.println(item);
        }
        System.out.println("--------------------------");
        System.out.println("Total de registros: " + lista.size());
    }

    public static void exibirTempoExecucao(int rotina, long tempoMs) {
        System.out.println("Tempo (ms) rotina " + rotina + ": " + tempoMs);
    }
}
